notebooks/COS/Setup/Setup.ipynb

```diff
@@ -0,0 +1,324 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"topS\"></a>\n",
+    "\n",
+    "# Setting up your environment\n",
+    "\n",
+    "# Learning Goals\n",
+    "### This Notebook is designed to walk the user (*you*) through:\n",
+    "#### 1. **[Setting up a `conda` environment for working with COS data](#setupEnvS)**\n",
+    "##### - 1.1. [Installing `conda`](#installcondaS)\n",
+    "##### - 1.2. [Creating a `conda` environment](#createenvS)\n",
+    "#### 2. **[Setting up the git repo of COS tutorials](#tutS)** (*optional*)\n",
+    "#### 3. **[Downloading up-to-date reference files](#crdsS)**\n",
+    "##### - 3.1. [Downloading the most recent context](#newcC)\n",
+    "##### - 3.2. [Downloading an older context](#oldcC)\n",
+    "\n",
+    "\n",
+    "# 0. Introduction\n",
+    "#### The Cosmic Origins Spectrograph ([*COS*](https://www.nasa.gov/content/hubble-space-telescope-cosmic-origins-spectrograph)) is an ultraviolet spectrograph on-board the Hubble Space Telescope ([*HST*](https://www.stsci.edu/hst/about)) with capabilities in the near ultraviolet (*NUV*) and far ultraviolet (*FUV*).\n",
+    "\n",
+    "#### This tutorial aims to prepare you to begin analyzing COS data of your choice by walking you through reading and viewing a spectrum obtained with COS, as well as obtaining a few diagnostic measurements of your spectrum.\n",
+    "\n",
+    "- For an in-depth manual to working with COS data and a discussion of caveats and user tips, see the [COS Data Handbook](https://hst-docs.stsci.edu/display/COSDHB/).\n",
+    "- For a detailed overview of the COS instrument, see the [COS Instrument Handbook](https://hst-docs.stsci.edu/display/COSIHB/).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Notes for those new to Python/Jupyter/Coding:\n",
+    "\n",
+    "- You will frequently see exclamation points (**\\!**) or dollar signs (**\\$**) at the beginning of a line of code. These are not part of the actual commands. The exclamation points tell a jupyter notebook to pass the following line to the command line, and the dollar sign merely indicates the start of a terminal prompt. \n",
```

Comment:
Not proposing any change right now, but I wonder if we should move this to some generic space since it's true of basically *all* notebboks?  Something to think about for the future.

Reply:
I think I might _also_ put it in the website portion (maybe a notes sub-section of the Requirements), but I think it's ok to repeat in the Setup notebook.
I guess it might be worth adding in a more general space visible to people looking at the _other non-COS_ notebooks

Reply:
Added this to the README which I also committed.